cranelift/assembler-x64/meta/src/instructions/cmp.rs

```diff
@@ -0,0 +1,34 @@
+use crate::dsl::{EflagsMutability::*, Feature::*, Inst, Location::*};
+use crate::dsl::{fmt, inst, r, rex, rw, sxl, sxq, sxw};
+
+#[rustfmt::skip] // Keeps instructions on a single line.
+pub fn list() -> Vec<Inst> {
+    vec![
+        inst("cmpb", fmt("I", [r(al), r(imm8)]), rex(0x3C).ib(), _64b | compat),
```

Comment:
We don't end up lowering to these instructions yet, right? If so, a simplification here could be to keep these instructions out of this PR and add them in once we can do the ISLE conversions at the same time. (I mean, we could also leave them in this PR since we know we'll eventually use them, I'm just flagging that they're not yet used).